### ISSUE TITLE
USWDS - Site - Known issues: Add known issues to memorable date component

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-site#2682]
Usability test findings from the 'zebra' batch, which included testing memorable date, revealed some issues that we should add to the 'known issues' section for memorable date. We should also add an alert.

Issues:

- [USWDS - Memorable date: Input formatting requirements are unclear to users of screen readers #5902](https://github.com/uswds/uswds/issues/5902)
- [USWDS - Memorable date: Dropdown search functionality doesn't meet some users' expectations. #5910](https://github.com/uswds/uswds/issues/5910)
- [USWDS - Memorable date: Dropdown options containing month name and number can be confusing for screen reader users. #5911](https://github.com/uswds/uswds/issues/5911)
